src/gen_rpc.app.src

```diff
@@ -5,25 +5,26 @@
 %%% Copyright 2021 Miniclip. All Rights Reserved.
 %%%
 
-{application, gen_rpc,
-    [{description, "A scalable RPC library for Erlang-VM based languages"},
+{application, gen_rpc, [
+    {description, "A scalable RPC library for Erlang-VM based languages"},
     {vsn, git},
+    {registered, []},
     {mod, {gen_rpc_app, []}},
-    {registered, [gen_rpc_dispatcher]},
-    {maintainers, ["Miniclip"]},
-    {licenses, ["Apache 2.0"]},
+    {pkg_name, mc_gen_rpc},
+    {licenses, ["Apache License, Version 2.0"]},
     {links, [{"Github", "https://github.com/miniclip/gen_rpc"}]},
-    {files, [
-        "LICENSE",
-        "README.md",
-        "CHANGELOG.md",
-        "include",
-        "package.exs",
-        "rebar.config",
-        "src"
+
+    {applications, [
+        % OTP dependencies
+        kernel,
+        stdlib,
+        crypto,
+        asn1,
+        public_key,
+        ssl
+        % External dependencies
```

Comment:
^ Missing `hut` and `ssl_verify_fun`

(I guess it was working out of sheer luck - likely neither dependency is required to be started in order to be used.)

Reply:
I can add, no prob.

Reply:
(hut isn't an app, neither is ssl_verify_fun)

Reply:
ca45044

Reply:
Well, `ssl_verify_fun` does depend on `ssl` to be started, but fortunately `gen_rpc` also does :smile:

Reply:
cool-i-o